packages/taquito/src/batch/rpc-batch-provider.ts

```diff
@@ -208,17 +191,6 @@ export class OperationBatch extends OperationEmitter {
     return this;
   }
 
-  /**
-   *
-   * @description Add a tx rollup batch operation to the batch
-   *
-   * @param params Tx rollup batch operation parameter
-   */
-  withTxRollupSubmitBatch(params: TxRollupBatchParams) {
-    this.operations.push({ kind: OpKind.TX_ROLLUP_SUBMIT_BATCH, ...params });
-    return this;
-  }
-
   async getRPCOp(param: ParamsWithKind) {
```

Comment:
reveal and updateConcensusKey are both manager operation which [can be batch](https://tezos.gitlab.io/nairobi/blocks_ops.html?highlight=update_consensus_key#manager-operation-batches) and activation is not (understand it's old code, just would be nice to correct it)

Reply:
yeah activation is not batchable, that was def copied over. Gonna update that

Reply:
it doesn't seem like updateConsensuskey is supported in batch. Iirc I had a conversation with Roxane to not batch it since it makes a bit less sense for users/bakers to put that into a batch operation. So maybe we can address that some other time?